src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,49 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    private function getBundleData()
+    {
+        $bundles = array();
+
+        $composerLockPath = $this->kernel->getRootDir().'/../composer.lock';
+        if (!file_exists($composerLockPath)) {
+            $installedPackages = array();
+        } else {
+            $composerLock = json_decode(file_get_contents($composerLockPath), true);
+            foreach (array_merge($composerLock['packages'], $composerLock['packages-dev']) as $package) {
```

Comment:
``packages-dev`` may be ``null`` if the lock file was generated with a ``--no-dev`` option